go.mod

```diff
@@ -4,30 +4,28 @@ go 1.18
 
 require (
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.0
+	github.com/onsi/gomega v1.23.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.56.3
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.56.3
 	github.com/prometheus/client_golang v1.12.1
 	github.com/prometheus/common v0.32.1
 	github.com/spf13/cobra v1.5.0
 	github.com/stretchr/testify v1.8.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.24.3
-	k8s.io/apimachinery v0.24.3
+	k8s.io/api v0.26.3
+	k8s.io/apimachinery v0.26.3
 	k8s.io/apiserver v0.24.3
```

Comment:
Could you please also update the following packages so that all the Kubernetes dependencies are on the same version:
- k8s.io/apiserver
- k8s.io/component-base
- k8s.io/metrics

Reply:
[c84e5ab](https://github.com/kubernetes-sigs/prometheus-adapter/pull/574/commits/c84e5ab4c618b639e7e46cea589c94a35f3116b1)